stock_available_immediately/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Camptocamp, Akretion, Numérigraphe
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Ignore planned receptions in quantity available to promise",
+    "version": "10.0.1.0.0",
+    "depends": ["stock_available"],
+    "author": "Camptocamp,Sodexis,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Hidden",
```

Comment:
Could you maybe add some missing entries like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py ?

Could you change category to 'Warehouse Management' as we can find this module more easily on OCA shop ?